test/common/markdown/markdown.tests.ps1

```diff
@@ -9,7 +9,7 @@ $repoRootPathFound = $false
 Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
     BeforeAll {
         # Skip if not windows, We don't need these tests to run on linux (the tests run fine in travis-ci)
-        $skip = !$IsWindows
+        $skip = $false
```

Comment:
Is this a temporary change, or should we just take the `if`-statement out?

Reply:
removed this change.  It was to allow local testing of the change.